frb_rust/Cargo.toml

```diff
@@ -78,5 +72,8 @@ chrono = ["dep:chrono", "allo-isolate/chrono"]
 wasm-start = ["console_error_panic_hook"]
 thread-pool = ["dep:threadpool"]
 rust-async = ["dep:tokio", "dep:futures", "dep:wasm-bindgen-futures"]
-user-utils = ["dep:android_logger", "dep:oslog"]
+user-utils = []
 dart-opaque = ["dep:dart-sys-fork"]
+
+[lints.rust]
+unexpected_cfgs = { level = "allow", check-cfg = ['cfg(wasm)'] }
```

Comment:
looks like we may not allow this. instead, let's migrate all `cfg(wasm)` to standard things like `#[cfg(target_family = "wasm")]` - maybe we can do this in another PR to keep things clear

Reply:
Done in #2329 and removed the lint.